bobweb/bob/scheduler.py

```diff
@@ -47,29 +55,34 @@ class Scheduler:
     Where the call back would be:
         async def good_morning_broadcast(self):
             await broadcaster.broadcast(self.updater.bot, "HYVÄÄ HUOMENTA!")
+
+
     """
-    def __init__(self, application: Application):
-        # First invoke all jobs that should be run at startup and then add recurrent tasks
-        # At the startup do broadcast and promote action immediately once
-        application.job_queue.run_once(broadcast_and_promote, 0)
-        application.job_queue.run_once(start_message_board_service, 5)
-
-        # Every day at 18:00:30
-        application.job_queue.run_daily(days=EVERY_WEEK_DAY,
-                                        time=datetime.time(hour=18, minute=0, second=30, tzinfo=fitz),
-                                        callback=daily_announce_new_free_epic_games_store_games)
-
-        # At 17:00 on Friday
-        application.job_queue.run_daily(days=FRIDAY,
-                                        time=datetime.time(hour=17, minute=0, tzinfo=fitz),
-                                        callback=friday_noon)
-
-        # Every midnight empy SahkoCommand cache
-        application.job_queue.run_daily(days=EVERY_WEEK_DAY,
-                                        time=datetime.time(hour=0, minute=0, tzinfo=fitz),
-                                        callback=nordpool_service.cleanup_cache)
-
-        logger.info("Scheduled tasks added to the job queue")
+
+    # First invoke all jobs that should be run at startup and then add recurrent tasks.
+    # Startup tasks are done after delay (in seconds) so that the bot has time to first start up
+    application.job_queue.run_once(broadcast_and_promote, 0, job_kwargs=default_job_props)
+    application.job_queue.run_once(start_message_board_service, 5, job_kwargs=default_job_props)
+
+    # Every day at 18:00:30
+    application.job_queue.run_daily(days=EVERY_WEEK_DAY,
+                                    time=datetime.time(hour=18, minute=0, second=30, tzinfo=fitz),
+                                    callback=daily_announce_new_free_epic_games_store_games,
+                                    job_kwargs=default_job_props)
+
+    # At 17:00 on Friday
+    application.job_queue.run_daily(days=FRIDAY,
+                                    time=datetime.time(hour=17, minute=0, tzinfo=fitz),
+                                    callback=friday_noon,
```

Comment:
A little confusing naming `friday_noon`. Maybe `end_of_workweek_greeting` instead, for example.

Reply:
Renamed to 'backup_with_end_of_work_week_greeting'.